tests/test_key_standardisation.cc

```diff
@@ -25,12 +25,25 @@
  * Boston, MA 02111-1307, USA.
  */
 
-#include <boost/test/unit_test.hpp>
-#include <boost/mpl/list.hpp>
-
 #include "utils/key_standardisation.hh"
 
+#include <boost/mpl/list.hpp>
+#include <boost/test/unit_test.hpp>
+
 namespace rascal {
+  /**
+   * Overload of stdout stream for tuple standardisation for debugging purposes
+   */
+  template <typename T, size_t Order>
+  std::ostream & operator<<(std::ostream & os,
+                            const KeyStandardisation<T, Order> & index) {
+    os << "(";
+    for (size_t i = 0; i < Order - 1; ++i) {
+      os << index[i] << ", ";
+    }
+    os << index.back() << ")";
+    return os;
```

Comment:
Well ok, here it is....Resolved I guess.

Reply:
Yes, I putted it where it was needed. This also removes the `#include <iostream>` from inside rascal main headers.